xml/schema/Pledge/Pledge.xml

```diff
@@ -249,8 +251,10 @@
     <comment>Date this pledge finished successfully (total pledge payments equal to or greater than pledged amount).</comment>
     <add>2.1</add>
     <html>
-        <type>Select Date</type>
+      <type>Select Date</type>
     </html>
+    <uniqueName>pledge_end_date</uniqueName>
+    <uniqueTitle>Payments Ended Date</uniqueTitle>
```

Comment:
OK this confused me - I think it should be pledge end date

Reply:
@eileenmcnaughton why? i thought unique title was for the search form but title would be the more standard pledge end date

Reply:
OK - I looks like it's unchanged in the  UI so perhaps it's just me that thought  pledge end made sense